IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/NewsCollectionView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NewsCollectionView.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+final class NewsCollectionView: UICollectionView, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    var items: [NewsArticleViewModel] = []
```

Comment:
We should use Bond's `MutableObservableArray< NewsArticleViewModel >` so we bind it